Source/Core/Core/Core.cpp

```diff
@@ -170,6 +170,11 @@ void DisplayMessage(std::string message, int time_in_ms)
   OSD::AddMessage(std::move(message), time_in_ms);
 }
 
+bool IsBooting()
+{
+  return (s_is_booting.IsSet() || !s_hardware_initialized);
```

Comment:
```suggestion
  return s_is_booting.IsSet() || !s_hardware_initialized;
```

Also, `WaitUntilDoneBooting` should be made to call this function.